lib/tasks/sync_checks.rake

```diff
@@ -28,7 +28,7 @@ namespace :sync_checks do
     end
 
     checker.add_expectation("document_type") do |content_item, edition|
-      content_item["document_type"] == edition.artefact.kind
+      content_item["document_type"] == format
```

Comment:
Will this not fail when we run a sync check for a document that has changed format type at some point in its' lifetime? I think that's what @whoojemaflip and I came across and why it was changed.